msdsl/expr/expr.py

```diff
@@ -78,7 +78,7 @@ def __rsub__(self, other):
         return (-self).__add__(other)
 
     def __neg__(self):
-        return -1.0*self
+        return -1*self
```

Comment:
Turns out the solution to the negation problem was very concise :-)  I don't remember exactly why I used ``-1.0``, but I think that may have been due to a previous limitation of **msdsl** that has since been resolved.

Reply:
that is a small fix ya xD.